src/models/Factor.js

```diff
@@ -58,6 +59,7 @@ function (Okta, Q, factorUtil, Util, Errors, BaseLoginModel) {
           'GOOGLE',
           'YUBICO',
           'FIDO',
+          'CUSTOM',
```

Comment:
Do we need provider? Since we are moving away from requiring provider?

Reply:
We would need it just so that there is a known set of values.